string command
float64 data
